### ISSUE TITLE
[FEAT] Turn down logs on testing

### ISSUE THREAD

**Issue by lucpeterson** [LLNL/merlin#310]
## 🚀 Feature Request

**What problem is this feature looking to solve?**
Testing log files dump a bunch of info: hard to parse out which ones pass/fail

**Describe the solution you'd like**
the "local" tests could be run with a lower log level, eg "merlin -lvl ERROR run --local blah blah blah"

**Describe alternatives you've considered**
We could keep the same log level and redirect the output, then if the test fails return the output

**Additional context**
Just to make it easier for developers to parse which tests pass / fail. I'm not sure how pytest will handle this.

**Comment by ben-bay** [comment]
#309 will help this somewhat by showing which condition failed at the top, even whilst not in `--verbose` mode. I'll spin up a branch that changes the log level.

**Comment by AlexanderWinterLLNL** [comment]
Pytest will only log errors in individual tests that fail\*, but when it fails, it returns: the stack trace, logs at all levels unless you pass a CLI flag at the PyTest invocation indicating you'd like a more/less severe level than... I want to say Warning...? It will follow whatever Exception/Error management you have (which there are a lot of simple features to proactively and safely examine your software's Error management). 

I actually gave a presentation on PyTest for my last project so people knew about all the whistles/bells/corner cases, would it be valuable to set aside some time for a workshop for the broader group where I run through that so people know how to write tests for PyTest?

\*(so, if you have a poorly structured test that can't fail, you won't get any diagnostic information, that info is captured and suppressed, which isn't great, so you need to make sure there are genuine fail states for your tests)
